site/client/owid.scss

```diff
@@ -972,30 +1002,23 @@ article.page .authors-byline .citation-note:hover {
             text-decoration: underline;
         }
     }
-}
 
-/* Weird hack to increase the mouseover area of the inline footnotes */
-a.ref sup::after {
-    overflow: hidden;
-    text-indent: -9999rem;
-    font-size: 0;
-    line-height: 0;
-    position: absolute;
-    top: 50%;
-    left: 50%;
-    -webkit-transform: translate(-50%, -50%);
-    -moz-transform: translate(-50%, -50%);
-    -ms-transform: translate(-50%, -50%);
-    -o-transform: translate(-50%, -50%);
-    transform: translate(-50%, -50%);
-    content: "increase click area";
-    width: 100%;
-    height: 100%;
-    min-width: 1.875rem;
-    min-height: 1.875rem;
-    z-index: $zindex-footnote;
-    border-radius: 100%;
+    .tippy-content {
+        max-height: 40vh;
```

Comment:
@MarcelGerber Why go for `40vh` here? Isn't it a bit low, even for mobile?

Reply:
The reasoning here is that Tippy can either place the tooltip above or below the ref.
If the ref is exactly in the middle of the screen, then Tippy has only around `50vh` above or below the ref, respectively, to place the tooltip in. For that reason we can't go much higher than `40vw` - 45 would probably be fine, but not much more.

Reply:
That's that scenario:
![Screenshot_20200527-125002](https://user-images.githubusercontent.com/2641501/83010841-2db9cf80-a019-11ea-81b1-e9255f8d4212.jpg)

Reply:
Ah, right!